seed/accounts/accounts.py

```
@@ -1,18 +1,31 @@
 from seed.accounts.buyer import seed_buyer
 from seed.accounts.licensee import seed_licensee
 from seed.accounts.module import seed_module
 from seed.accounts.seller import seed_seller
 from seed.accounts.user_group import seed_user_group
+from seed.helper import init_resource
 
 logger = logging.getLogger(__name__)
 
 
-async def seed_accounts() -> None:  # noqa: WPS217
+async def get_account() -> Account:  # noqa: RUF029 async for compatibility purposes with init_resource
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `RUF029`)

Remove unused `noqa` directive

(RUF100)